main/resource-access.bs

```diff
@@ -75,6 +75,352 @@ Issue: Write Linked Data Platform section.
 Draft:
 A Solid data pod MUST conform to the LDP specification [[!LDP]].
 
+## Auxiliary Resources ## {#rm}
+
+### Background and Need ### {#ar-need}
+<em>This section is non-normative.</em>
+
+An auxiliary resource may provide supplementary information about a given
+Solid resource, or affect how that resource and others associated with it
+are processed, served, or interpreted. Different auxiliary resource types
+provide different capabilities. This section introduces a mechanism for linking
+auxiliary resources with regular Solid resources.
+
+Auxiliary resources are needed to influence the configuration, processing, or
+interpretation of Solid resources without changing the composition of the
+resources themselves. To do so would be undesirable in many cases, and not
+possible in others. Auxiliary resources are not meant to replace the ability
+of a Solid resource to self-describe.
+
+Examples of auxiliary resources in use include:
+
+<ul>
+  <li>A binary JPEG image linked to an auxiliary resource that includes
+  information describing that binary JPEG.</li>
+  <li>A container linked to an auxiliary resource that includes access control
+  statements for that container and the resources that belong to it.</li>
+  <li>A resource representation whose shape is constrained by a given ShEx
+  schema that links to an auxiliary resource defining that schema.</li>
+  <li>A resource with an associated set of configurable parameters links to an
+  auxiliary resource where those configurable parameters reside.</li>
+</ul>
+
+### Required Server-side Implementation ### {#ar-server}
+
+
+For any defined auxiliary resource available for a given Solid resource, all
+representations of that resource MUST include an HTTP `Link` header pointing to
+the location of each auxiliary resource.
+
+The `rel={relation-type}` [[!RFC8288]] will define the relationship to the
+target URL in the HTTP `Link` header. URIs are encouraged to indicate Link
+relation types.
+
+An auxiliary resource linked with a given Solid resource through an HTTP `Link`
+header is considered to be *directly associated* with that resource. It is up
+to the server to determine how that association affects processing based on the
+auxiliary resource type.
+
+A given Solid resource MAY link to zero or more auxiliary resources. A given
+Solid resource MAY Link to auxiliary resources on a different server under a
+different authority, per the configuration of the Solid server on which that
+resource resides.
```

Comment:
This entails that an ACL resource can be on a different server.. and so raises security issues that should be addressed. Are there existing issues tackling some of them? If the following is intended for WAC to address, that's fine (or if this text is written with known answers to the questions below, that's fine too).. but I would suggest that what's expected from ACLs should be aligned as opposed to assuming that it will work out somehow.

* Will agent(s) controlling a resource on server-A be authorized to request write operations on the ACL on server-B? Ditto the application.

* What should happen if/when remote ACL becomes inaccessible (for whatever reason for any period)?

* What should happen if/when remove ACL is compromised?

Reply:
I think that we can actually remove this statement, and even be explicit that we do not permit it. If any use cases present themselves that would force us to relax it, we can reevaluate. I've gone back and looked at the genesis of this, and don't see a good reason beyond an attempt for added flexibility in the future.

Reply:
This has been [updated](https://github.com/solid/specification/pull/156/commits/4b48d5fc42273789e07c20c853b5bf4592641aff) to constrain auxiliary resources to the same server / authority, with issue #176 noted inline to determine whether MUST NOT is appropriate.